packages/kbn-securitysolution-url-hooks/README.md

```diff
@@ -0,0 +1,3 @@
+# @kbn/securitysolution-url-hooks
+
+Empty package generated by @kbn/generate
```

Comment:
When you have time, could you add a read me, even if just a brief one?

Reply:
done